### PR TITLE
feat: Add Virtual Box setup instructions

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Rough Virtual Box setup

# Changelog

Add README and setup scripts

# Review requests

Try out setting up a machine

# Risk assessment

None

### CODE REVIEW

File: virtual-box-setup.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Virtual Box Setup
+
```
Comment (amitlissack) [inline virtual-box-setup.md:2]: Does it matter which iso file I get? Could a link be provided?

Comment (DerekMaggio) [reply virtual-box-setup.md:2]: Updated

File: virtual-box-setup.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+   14. Wait for installation and all updates to finish. 
+       1. When finished it will say Reboot Now. 
+7. Remove Installation Media
+   1. Go back to the Virtual Box window
+   2. Select your Virtual Machine
+   3. Click Settings
+   4. Click Storage
+   5. Delete the optical disk you installed earlier
+8. Reboot 
+    1. Go back to the Virtual Machine command line and select Reboot Now
```
Comment (amitlissack) [inline virtual-box-setup.md:70]: command line?

**Review by amitlissack — APPROVED** [review]
Verified that this works. Thank!

**Pull request merged**